packages/grid_client/README.md

```diff
@@ -51,7 +51,7 @@ yarn install
 
 - Network environment: should select dev environment, qa, test or main.
 
-- Mnemonic: 12 words for your account. [create one](https://manual.grid.tf/getstarted/TF_Dashboard/TF_Dashboard.html#create-polkadot-extension-account)
+- Mnemonic: 12 words for your account. [create one](https://www.manual.grid.tf/documentation/dashboard/wallet_connector.html)
```

Comment:
The link should redirect to create an account not the wallet connector. I know we're not using Polkadot extension anymore so maybe as a fix, we can rephrase this section accordingly.

Reply:
rephrased to profile and wallet connector. good catch